pom.xml

```diff
@@ -22,7 +22,7 @@
       (for scm, site-distribution, etc -->
     <project_name>fcrepo</project_name>
     <project_organization>fcrepo</project_organization>
-    <fcrepo-storage-ocfl.version>6.5.0-SNAPSHOT</fcrepo-storage-ocfl.version>
```

Comment:
If we do this, then when you make a change to `fcrepo-storage-ocfl` locally and build it, you will need to remember to change this variable to `7.0.0-SNAPSHOT` to test it.

Also, if we are doing this perhaps we should be adding a github action to `fcrepo-storage-ocfl` that deploys `fcrepo/fcrepo:main`, changes the `fcrepo-storage-ocfl.version` to whatever the current version is and tries to run the Fedora tests?